Source/SwiftLintFramework/Rules/OpeningBraceRule.swift

```diff
@@ -9,7 +9,21 @@
 import Foundation
 import SourceKittenFramework
 
-public struct OpeningBraceRule: Rule {
+private let violatingPattern = "((?:[^( ]|[\\t\\n\\f\\r (] )\\{)"
+private let correctString = " {"
```

Comment:
since this is only used in `OpeningBraceRule.writeToFile()`, it could just be inlined into that method.